graylog2-server/src/test/java/org/graylog2/inputs/routing/InputRoutingServiceTest.java

```diff
@@ -156,6 +168,20 @@ void handleInputRenamed() throws IOException, NotFoundException {
         verify(ruleService, times(1)).save(any(), anyBoolean());
     }
 
+    @Test
+    void handleStreamRenamed() throws IOException, NotFoundException {
+        final RuleDao ruleDao = loadFixture("org/graylog2/inputs/routing/InputRoutingRule1.json", RuleDao.class);
+        when(ruleService.loadAllByTitle(anyString())).thenReturn(List.of(ruleDao));
+
+        String pipelineSource = "pipeline \"Default Routing\"\nstage 0 match EITHER\nrule \"gl_route_inputName_to_streamName\"\nend";
+        when(pipelineService.loadByName(anyString())).thenReturn(PipelineDao.builder().id("pipelineId1").title("pipelineDao").source(pipelineSource).build());
+
+        inputRoutingService.handleStreamRenamed(new StreamRenamedEvent(STREAM_ID, STREAM_NAME, STREAM_NEW_NAME));
+
+        verify(ruleService).save(argThat(ruleDao1 -> ruleDao1.source().contains(STREAM_NEW_NAME)), anyBoolean());
+        verify(ruleService, times(1)).save(any(), anyBoolean());
```

Comment:
If `times(x)` is not specified, it defaults to 1.
```suggestion
        verify(ruleService).save(any(), anyBoolean());
```

Reply:
fixed